velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -770,7 +770,17 @@ uint32_t HiveDataSink::appendWriter(const HiveWriterId& id) {
   if (!options->zstdCompressionLevel) {
     options->zstdCompressionLevel =
         compressionLevel.value_or(kDefaultZstdCompressionLevel);
-  }
+
+    auto parquetDataPageVersion =
+        hiveConfig_->parquetDataPageVersion(connectorSessionProperties);
+
+    if (parquetDataPageVersion == "PARQUET_1_0") {
+      options->parquetDataPageVersion =
+          dwio::common::ParquetDataPageVersion::V1;
+    } else {
```

Comment:
Check if it's "PARQUET_2_0", otherwise throw an error

Reply:
My thinking was that `parquetDataPageVersion` value must be either "PARQUET_1_0" or "PARQUET_2_0" if we've reached this point, as I added validation in the `parquetDataPageVersion()` function from which this value is fetched:

```
VELOX_CHECK(
      parquetDataPageVersion == "PARQUET_1_0" ||
          parquetDataPageVersion == "PARQUET_2_0",
      "Invalid Parquet version.");
```